app/collins/models/shared/IpAddressable.scala

```diff
@@ -127,11 +127,11 @@ trait IpAddressStorage[T <: IpAddressable] extends Schema with AnormAdapter[T] w
   * For a range 0L..20L, used addresses List(17,18,19,20), the result will be None (allocate from beginning)
   */
   protected def getCurrentLowestLocalMaxAddress(calc: IpAddressCalc)(implicit scope: Option[String]): Option[Long] = inTransaction {
-    val minAddress = calc.minAddressAsLong
+    val startAddress = calc.startAddressAsLong
```

Comment:
It could be since the default for the gateway and startAddress are both minAddress.
If there is no gateway on the network or the gateway is elsewhere, this would be correct, but people should configure startAddress properly unless that is true.